novawallet/Modules/NetworkDetails/NetworkDetailsInteractor.swift

```diff
@@ -166,6 +197,93 @@ private extension NetworkDetailsInteractor {
             nodesConnections[node.url] = connection
         }
     }
+
+    func disconnectNodes() {
+        nodesConnections.values.forEach { connection in
+            connection.disconnect(true)
+        }
+
+        nodesConnections = [:]
+    }
+
+    func measureNodePing(for nodeUrl: URL) {
```

Comment:
Let's at least move ping logic to the operation factory. So, the NodePingOperationFactory will have a method to create ping operation for a given chain and node